laterpay/asset_sources/js/laterpay-backend-options.js

```diff
@@ -0,0 +1,64 @@
+(function($) {$(function() {
+
+    // encapsulate all LaterPay Javascript in function laterPayBackendAccount
+    function laterPayBackendOptions() {
+        var $o = {
+                // API credentials
+                browscapCacheUpdateButton       : $('.lp_js_BrowscapCacheUpdate'),
```

Comment:
it's only one, thus it should better use an ID

The rule of only using classes only applies to CSS, because of specificity reasons.